lib/masdb/register.ex

```diff
@@ -46,6 +46,18 @@ defmodule Masdb.Register do
   }
   defstruct [stores: [], schemas: [], tables: []]
 
+  def merge_schemas(olds, news) do
+    olds ++ news
+    |> Enum.group_by(fn s -> s.name end)
+    |> Enum.to_list
+    |> choose_schemas([])
+  end
+  defp choose_schemas([], acc), do: Masdb.Schema.sort(acc)
+  defp choose_schemas([{_, schemas}|tail], acc) do
+    schema = Enum.min_by(schemas, fn s -> {s.creation_time, s} end)
```

Comment:
Not 100% sure, but

> This lets older schema overwrite a newer schema. We can do this since our schemas are immutable. I'm not 100% sure, but I think this could happen when a new schema is denied.